packages/kubernetes/changelog.yml

```
@@ -1,4 +1,12 @@
 # newer versions go on top
+- version: 1.62.0
+  changes:
+    - description: |
+        [Breaking changes included] Make Kubernetes integration settings resource centric. This version includes breaking changes!!! 
+        Configuration options of kubernetes policy might be lost during the upgrade from previous versions. 
+        Specific options might need to be reverted manually to previous values if users want to keep same configuration after upgrading to this version.
+      type: enhancement
```

```suggestion
      type: breaking-change
```
looks like you can set it as a `breaking-change` https://github.com/elastic/package-spec/blob/main/spec/integration/changelog.spec.yml#L30

---

Done